server/src/sv_main.cpp

```diff
@@ -1165,6 +1165,15 @@ bool SV_AwarenessUpdate(player_t &player, AActor *mo)
 
 		return true;
 	}
+	else if (previously_ok && ok 
+			&& player.mo && !mo->player && mo->type != MT_AVATAR && mo->type != MT_PLAYER 
+			&& mo->on_conveyor
+			&& (gametic + mo->netid) % 120) // Update items on conveyers every 120 tics
```

Comment:
Is the client predicting the position of items on conveyors and this is just to ensure sync? ~3.5 seconds is a long time for an item to be on a conveyor.

Reply:
Seems to be a timing issue that affects more than just this (the item on the client is almost always behind, and it depends on when you connect/when the item respawns for how behind it is), this is just a rubber band to get it to work at QuakeCon. 😅